.devcontainer/docker-compose-local.yaml

```diff
@@ -73,27 +76,60 @@ services:
     ports:
       - 80:80
     environment:
+      # 
```

Comment:
remove?

Reply:
This would hold the description of `DEPLOYMENT_TYPE`. Right now it's just `local`, right?